template_config.yml

```diff
@@ -0,0 +1,42 @@
+# This config represents the latest values used when running the plugin-template. Any settings that
+# were not present before running plugin-template have been added with their default values.
+
+all: False
+bootstrap: False
+docs: False
+exclude_check_commit_message: False
+exclude_coverage: True
+exclude_deploy_client_to_pypi: False
+exclude_deploy_client_to_rubygems: False
+exclude_deploy_daily_client_to_pypi: False
+exclude_deploy_daily_client_to_rubygems: False
+exclude_deploy_to_pypi: False
+exclude_docs_test: False
+exclude_mariadb_test: False
+exclude_test_bindings: False
+plugin_camel: PulpDocker
+plugin_camel_short: Docker
+plugin_caps: PULP_DOCKER
+plugin_caps_short: DOCKER
+plugin_dash: pulp-docker
+plugin_dash_short: docker
+plugin_name: docker
+plugin_snake: pulp_docker
+plugin_snake_short: docker
+pypi_username: pulp
+test: False
+travis: True
+verbose: False
+black: False
```

Comment:
using black caused some problems, so I disabled it for now. We can add in another PR.